lock-api/src/main/java/com/palantir/lock/client/LockRefresher.java

```diff
@@ -70,8 +70,10 @@ private void refreshLocks() {
             if (!refreshFailures.isEmpty()) {
                 log.info("Failed to refresh {} lock tokens, most likely because they were lost on the server."
                                 + " The first (up to) 20 of these were {}.",
+                        SafeArg.of("successfullyRefreshed", successfullyRefreshedTokens.size()),
```

Comment:
LargeInternalProduct will probably be sad on its internal checkstyle, since the number of `{}` won't line up - please add one to the beginning of the message i.e. `Successfully refreshed {} lock tokens, but failed to refresh {} ...

Reply:
Fixed